docs/snippets/first.nf

```diff
@@ -15,5 +15,5 @@ Channel.of( 1, 2, 'a', 'b', 3 )
 
 // emits the first item for which the predicate evaluates to true: 4
 Channel.of( 1, 2, 3, 4, 5 )
-    .first { it > 3 }
+    .first { v -> v > 3 }
```

Comment:
While I agree it's generally better to use the extended syntax. it would be nice to mention about `it` as implicit argument and the possibility to have one more argument in a closure, when describing what a closure is: 


```
A closure is a function that can be used like a regular value. Typically, closures are passed as arguments to higher-order functions to express computations in a declarative manner.

For example:

def square = { v -> v * v }
The above example defines a closure, which takes one parameter named v and returns the “square” of v (v * v). The closure is assigned to the variable square.
```

Reply:
It is mentioned at the bottom of the syntax page:

```md
## Deprecations

The following legacy features were excluded from this page because they are deprecated:

...
- The implicit `it` closure parameter
```

The migration notes for implicit `it` are covered in the private docs PR

Reply:
Not sure, `it` should be considered deprecated. worth noting there are also slight different semantic with it. I think it should be mentioned as a note or tip also because it's widely adopted, and therefore it would be informative to know about it

Reply:
Disagree

Reply:
Thought about this more. For the sake of new users reading existing code, I have added a note about the implicit closure parameter to the Scripts page. There is also an example of a closure with multiple parameters there.